lib/storage/data/file/DataFileStore.js

```diff
@@ -165,7 +169,8 @@ class DataFileStore {
                           { method: 'put', key, filePath, error: err });
                 // destroying the write stream forces a close(fd)
                 fileStream.destroy();
-                return cbOnce(err);
+                return cbOnce(errors.InternalError.customizeDescription(
+                    `write stream error: ${err}`));
```

Comment:
Is this an object of Error instance? If yes, wouldn't this be an `{}` if you stringify it?